### PR TITLE
Port to additional platforms

### PR DESCRIPTION
**Pull request by ilanschnell**
The source URL was giving many problems on different build machines.  I found an alternative URL with the source tarball, and now the builds are working everywhere.

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
the package seems to be missing the `dev_url` and `doc_url`
http://gerris.dalembert.upmc.fr/darcs/gts-stable/

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,19 +31,28 @@ requirements:
...
-    - test -f ${PREFIX}/lib/libgts.so  # [linux]
-    - test -f ${PREFIX}/lib/libgts.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libgts.so                     # [linux]
+    - test -f ${PREFIX}/lib/libgts.dylib                  # [osx]
     - if not exist %LIBRARY_PREFIX%\\bin\\gts.dll exit 1  # [win]
 
 about:
   home: http://gts.sourceforge.net/
-  license: LGPL-2.0-or-later
+  license: GPL verision 2
```
Comment (tobijk) [inline recipe/meta.yaml]: This is not a valid SPDX license identifier, plus it is misspelled.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
       - 0001-Disable-USE_PREDICATES_INIT.patch  # [build_platform != target_platform]
   - path: CMakeLists.txt
 
 build:
-  number: 2
+  number: 3
+  skip: True    # [win32]
   run_exports:
-    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ pin_subpackage('gts', max_pin='x.x') }}
```
Comment (tobijk) [inline recipe/meta.yaml:17]: Was there a reason this had to be made explicit?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: gts
+  version: 0.7.6
 
 source:
-  - url: http://downloads.sourceforge.net/gts/gts-{{ version }}.tar.gz
+  - url: http://iweb.dl.sourceforge.net/project/gts/gts/0.7.6/gts-0.7.6.tar.gz
```
Comment (tobijk) [inline recipe/meta.yaml]: Could we make this 

    http://iweb.dl.sourceforge.net/project/gts/gts/{{ version }}/gts-{{ version }}.tar.gz

?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,22 +1,20 @@
-{% set name = "gts" %}
-{% set version = "0.7.6" %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: gts
+  version: 0.7.6
```
Comment (tobijk) [inline recipe/meta.yaml:3]: Is there a particular reason, why the (name and) version had to be made explicit here? Consider making at least the version a template variable.

**Review by tobijk — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 5]
```diff
@@ -33,19 +31,28 @@ requirements:
...
     - if not exist %LIBRARY_PREFIX%\\bin\\gts.dll exit 1  # [win]
 
 about:
   home: http://gts.sourceforge.net/
-  license: LGPL-2.0-or-later
+  license: GPL verision 2
   license_file: COPYING
-  summary: |
-    GNU Triangulated Surface Library
+  license_family: GPL2
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: If `license` is `LGPL-2.0-or-later` so `license_family` should be `LGPL`

File: recipe/meta.yaml (hunk 1 above)
Comment (ilanschnell) [reply recipe/meta.yaml]: I fixed the typo.  I wasn't aware that the `license` field has to conform with the SPDX license identifier.  My understanding was that `license` is a free format field, and `license_family` has to be one of the strings listed here: https://github.com/conda/conda-verify/blob/master/conda_verify/constants.py#L8

File: recipe/meta.yaml (hunk 2 above)
Comment (ilanschnell) [reply recipe/meta.yaml:17]: Quote from the Zen of Python: `Explicit is better than implicit.`

File: recipe/meta.yaml (hunk 3 above)
Comment (ilanschnell) [reply recipe/meta.yaml]: I was having trouble with the download link (and some build workers still do), so I made it explicit which is easier for testing.

File: recipe/meta.yaml (hunk 5 above)
Comment (ilanschnell) [reply recipe/meta.yaml]: Thanks, I'll fix it.

File: recipe/meta.yaml (hunk 4 above)
Comment (katietz) [reply recipe/meta.yaml:3]: yeah, that looks to me not really required, even if this is more nit

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:17]: fair enough

**Review by katietz — APPROVED** [review]
thanks

**Pull request merged**